### PR TITLE
Feature: Include individual site summaries in the aggregated report of a sitemap

### LINKED ISSUE #850: Enhance Aggregated Report CSV with URL Column
## Feature Request

**Description:**
Enhance the existing aggregated `report.csv` report generated by the PSAT CLI to include the URL for each row of cookie data.

**Motivation:**
Currently, the aggregated `report.csv` file provides a summary of cookie counts across all scanned URLs. However, it lacks the context of which URL each row of data corresponds to. This makes it difficult to analyze and compare cookie usage across different websites directly from the aggregated report. 

**User Story:**
As a PSAT CLI user analyzing aggregated reports, I want to see the corresponding URL for each row of cookie data so that I can easily compare cookie usage across multiple websites within a single view.

**Acceptance Criteria:**
- [ ] The downloaded aggregated `report.csv` file includes a "URL" column as the first column.
- [ ] Each row in the `report.csv` file contains the URL of the website the cookie data belongs to.
- [ ] The order of columns to be 
  1. URL
  2. Total Cookies
  3. First Party
  4. Third Party
  5. Analytics
  6. Functional
  7. Marketing
  8. Uncategorized
  9. Cookies Issues
  10. Analytics Issues
  11. Functional Issues
  12. Marketing Issues
  13. Uncategorized Issues

**Additional Information:**
The current format of the aggregated .csv file is:
```
Total Cookies, 34
Total First Party Cookies, 32
Total Third Party Cookies, 2
...
Uncategorized Cookies with Issues, 8
```

The desired format after this feature implementation should be:
```
URL,Total Cookies,First Party,Third Party,Analytics,Functional,Marketing,Uncategorized,Cookies Issues,Analytics Issues,Functional Issues,Marketing Issues,Uncategorized Issues
"Aggregated",46,36,5,2,4,4,24,8,0,0,0,8
"https://example.com",34,32,2,2,4,4,24,8,0,0,0,8
"https://another-example.com",12,10,2,1,3,1,7,3,0,1,0,2 
...
```
This change will significantly improve the usability of the aggregated .csv report for analysis and comparison purposes.

### PR DESCRIPTION
**Pull request by amovar18**
## Description
The PR aims to modify the generated CSV for a sitemap, where the count for each URL is provided along with the aggregated counts.

## Testing Instructions
- Clone this PR.
- In the terminal run `npm i` && `npm run build:cli`.
- Now run analysis on a sitemap with `-o` or download the zip from the CLI-dashboard.

---

## Checklist

- [x] I have thoroughly tested this code to the best of my abilities.
- [x] I have reviewed the code myself before requesting a review.
- [x] This code is covered by unit tests to verify that it works as intended.
- [x] The QA of this PR is done by a member of the QA team (to be checked by QA).

Fixes #850

### CODE REVIEW

File: packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts [hunk 1]
```diff
@@ -107,27 +100,62 @@ const generateRootSummaryDataCSV = (
...
+  return summary.join(',').concat('\r\n');
+};
+export const generateRootSummaryDataCSV = (
+  siteMapAnalysisData: CompleteJson[]
+) => {
+  const extractedData: { [key: string]: CookieTableData } = reshapeCookies(
+    extractReportData(siteMapAnalysisData).landingPageCookies
+  );
+  const headers = [
+    'URL Heading',
```
Comment (gagan0123) [inline packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts:127]: Why `URL Heading` instead of just `URL` ?

Comment (amovar18) [reply packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts]: Fixed in latest commit. [dd317e1](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/849/commits/dd317e1c77f2549884b08f295799731b642062df)

**Comment by milindmore22** [comment]
QA : The heading for "Analytics Cookies with issues, Functional cookies with issues, Marketing Cookies with issues, Uncategorized cookies with issues" are not correct in CSV.

File: packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts [hunk 2]
```diff
@@ -107,27 +100,62 @@ const generateRootSummaryDataCSV = (
...
+
+  return summary.join(',').concat('\r\n');
+};
+export const generateRootSummaryDataCSV = (
+  siteMapAnalysisData: CompleteJson[]
+) => {
+  const extractedData: { [key: string]: CookieTableData } = reshapeCookies(
+    extractReportData(siteMapAnalysisData).landingPageCookies
+  );
+  const headers = [
```
Comment (mayan-000) [inline packages/common/src/utils/generateReports/generateRootSummaryDataCSV.ts:126]: Many of heading texts have translations available.

**Review by mohdsayed — APPROVED** [review]
Looks good.

**Pull request merged**